.gitmodules

```diff
@@ -1,18 +1,18 @@
 [submodule "src/coreclr"]
 	path = src/coreclr
 	url = https://github.com/dotnet/coreclr
-	branch = release/2.0.0
+	branch = .
 [submodule "src/roslyn"]
 	path = src/roslyn
 	url = https://github.com/dotnet/roslyn
 [submodule "src/core-setup"]
 	path = src/core-setup
 	url = https://github.com/dotnet/core-setup
-	branch = release/2.0.0
+	branch = .
 [submodule "src/standard"]
 	path = src/standard
 	url = https://github.com/dotnet/standard
-	branch = release/2.0.0
+	branch = .
```

Comment:
What does `.` do vs. deleting the line? (I wouldn't question just deleting these lines.)

Reply:
I believe this would mean the remote tracking branch would be copied from the current source-build branch, e.g. if you're on release/2.1 the corefx submodule's tracking branch would also be release/2.1.  I think it's only used for `submodule update --remote` which I don't think we very rarely use, so I'm fine either way on deleting vs. using `.`.

Reply:
Nifty. Yeah, we've never used that to any effect. No objection to it if @am11 finds it useful though, doesn't hurt anything.

Reply:
Oh. I just noticed it's described in the PR description. My bad. 😄

Reply:
Yup, and this way we do not need to update the `.gitmodules` file for each release.